### PR TITLE
feat(composer): generate library

### LINKED ISSUE #7869: Cloud Composer API: `composer.googleapis.com`

### PR DESCRIPTION
**Pull request by dbolduc**
Fixes #7869

Before you ask, changes to the scaffolding to drop the " API" suffix are coming in a separate PR.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7954)

### CODE REVIEW

File: google/cloud/composer/BUILD.bazel [hunk 1]
```diff
@@ -38,7 +38,7 @@ cc_library(
     deps = [
         "//google/cloud:google_cloud_cpp_common",
         "//google/cloud:google_cloud_cpp_grpc_utils",
-        "@com_google_googleapis//google/cloud/orchestration/airflow/service/v1:composer_cc_grpc",
```
Comment (coryan) [inline google/cloud/composer/BUILD.bazel]: Are you missing a dependency here?

**Review by coryan — CHANGES_REQUESTED** [review]
I think we want both `*_cc_grpc` rules in the `BUILD.bazel` file?

File: ci/etc/expected_install_directories [hunk 2]
```diff
@@ -68,10 +71,13 @@
...
 ./include/google/cloud/logging/internal
 ./include/google/cloud/logging/mocks
+./include/google/cloud/orchestration
+./include/google/cloud/orchestration/airflow
+./include/google/cloud/orchestration/airflow/service
+./include/google/cloud/orchestration/airflow/service/v1
 ./include/google/cloud/orgpolicy
 ./include/google/cloud/orgpolicy/v2
 ./include/google/cloud/oslogin
-./include/google/cloud/oslogin/common
```
Comment (dbolduc) [inline ci/etc/expected_install_directories:74]: Looks like I botched the rebase?

Comment (coryan) [reply ci/etc/expected_install_directories:74]: No, I think regenerating the libraries at `HEAD` removes that entry.  I suspect that the scaffold step and the `generate-libraries-pr` produce different outputs or something like that.

Comment (dbolduc) [reply ci/etc/expected_install_directories]: #7962

File: google/cloud/composer/BUILD.bazel (hunk 1 above)
Comment (dbolduc) [reply google/cloud/composer/BUILD.bazel]: No. There is only one `*_cc_grpc` rule.

**Comment by dbolduc** [comment]
This build hates me. I am just going to rebase and force push.

**Review by coryan — APPROVED** [review]

**Pull request merged**